bin/generate-shell-css.js

```diff
@@ -0,0 +1,29 @@
+var http = require( 'http' );
+var critical = require( 'critical' );
+var jade = require( 'jade' );
+
+// Stub vars and render shell HTML.
+var renderVars = {
+	head: {
+		metas: [],
+		links: []
+	},
+	renderedLayout: false,
+	hasSecondary: true,
+	urls: {}
+};
+
+var html = jade.renderFile( 'server/pages/index.jade', renderVars );
+
+// Generate the CSS required to render the initial shell.
+critical.generate( {
+	folder: 'public',
+	html: html,
+	css: [ 'public/style.css' ],
+	width: 1300,
+	height: 900,
+	dest: 'public/shell.css',
```

Comment:
@nb The `SHELL_CSS_PATH` constant is defined in `server/pages/index.js` currently, so it's not accessible here.

I could probably extract that constant out into its own file to be `require`'d here, and in `server/pages/index.js`. Maybe something like `server/pages/constants.js`? I could move some of the other constants over there too.

It would be nice to remove this duplication. Also on line 22, where we have `public/style.css`. We could find a way to not have that duplicated as well.

What do you think?